src/careamics/config/configuration_factory.py

```diff
@@ -243,20 +243,24 @@ def create_n2n_configuration(
     use_augmentations: bool = True,
     independent_channels: bool = False,
     loss: Literal["mae", "mse"] = "mae",
-    n_channels: int = 1,
+    n_channels_in: int = 1,
+    n_channels_out: int = -1,
     logger: Literal["wandb", "tensorboard", "none"] = "none",
     model_kwargs: Optional[dict] = None,
 ) -> Configuration:
     """
-    Create a configuration for training Noise2Noise.
+    Create a configuration for training CARE.
```

Comment:
Has this been changed to CARE from Noise2Noise because of the relaxation in constraints? Is CARE the umbrella term for supervised algorithms and Noise2Noise specifically uses a UNet? (This is just a question for my benefit).

Reply:
Argh, that's a mistake! I probably copied the whole docstring, rather than the just the parameters.

CARE = any supervised image restoration in microscopy (the benefit of being the first paper to do it at scale for scientific bioimages), actually the original principle is really any method that leverages structural priors for image restoration (so not supervised could also be argued). We have a tendency to use it for the supervised part.

Noise2Noise = self-supervised denoising by training noisy image to noisy image supervision

Reply:
Ok thanks for the clarification! That is what I had originally thought :)